services/src/main/resources/conf/platform.properties

```
@@ -12,7 +12,8 @@ org.exoplatform.calendar=@addon.exo.calendar.version@
 org.exoplatform.forum=@addon.exo.forum.version@
 org.exoplatform.wiki=@addon.exo.wiki.version@
 org.exoplatform.ecms=@addon.exo.ecms.version@
-org.exoplatform.social=@org.exoplatform.social.version@
+org.exoplatform.social=@io.meeds.distribution.version@
+io.meeds.social=@io.meeds.social.version@
```

Seems missing another property, same in `platform-private-distribution` (missing property).
See it here:
https://github.com/exoplatform/platform-public-distributions/blob/a48bdb378baec75dffc779af2e41df32e22c3cb5/services/src/main/resources/conf/platform.properties#L17

```suggestion
io.meeds.social=@io.meeds.social.version@
io.meeds.distribution=@io.meeds.social.version@
```